x-pack/plugins/encrypted_saved_objects/server/saved_objects/encrypted_saved_objects_client_wrapper.ts

```diff
@@ -263,6 +265,19 @@ export class EncryptedSavedObjectsClientWrapper implements SavedObjectsClientCon
     return await this.options.baseClient.closePointInTime(id, options);
   }
 
+  public createPointInTimeFinder(
+    findOptions: SavedObjectsCreatePointInTimeFinderOptions,
+    dependencies?: SavedObjectsCreatePointInTimeFinderDependencies
+  ) {
+    return this.options.baseClient.createPointInTimeFinder(findOptions, {
+      find: this.find.bind(this),
+      openPointInTimeForType: this.openPointInTimeForType.bind(this),
+      closePointInTime: this.closePointInTime.bind(this),
+      // Include dependencies last so that subsequent SO client wrappers have their settings applied.
+      ...dependencies,
```

Comment:
Since this wrapper is the highest priority over all the others, I guess that means we don't technically _have_ to allow the optional `dependencies` to override everythin. However, it felt safer to do it this way in case the wrapper ordering is changed in the future.